packages/logs/README.md

```diff
@@ -15,7 +15,7 @@ With the `datadog-logs` library, you can send logs directly to Datadog from JS c
 
 **Datadog client token**: For security reasons, [API keys][1] cannot be used to configure the `datadog-logs` library, because they would be exposed client-side in the JavaScript code. To collect logs from web browsers, a [client token][2] must be used. See the [client token documentation][2] for more details.
 
-**Datadog browser log library**: Configure the library through [NPM](#npm) or use the [bundle](#bundle) directly in the head tag.
+**Datadog browser log library**: Configure the library through [NPM](#npm) or use the [CDN async](#cdn-async) or [CDN sync](#cdn-sync) code snippets in the head tag.
```

Comment:
what about getting rid of all bundle references then?

Reply:
In the normal doc you mean?

Reply:
in this page at least, in other doc would be great as well

Reply:
My latest commit should have removed the last reference of bundle :)